pkg/eventhandler/eventhandlers.go

```diff
@@ -131,7 +136,11 @@ func (eh *EventHandler) startK8sJob(k8s k8sutils.K8s, action *config.Action, jso
 			}
 		}()
 
-		jobErr := k8s.AwaitK8sJobDone(jobName)
+		maxPollCount := defaultMaxPollCount
+		if task.MaxPollDuration != nil {
+			maxPollCount = *task.MaxPollDuration / pollIntervalInSeconds
```

Comment:
I guess a `ceil` would be better than a `floor` here.

Reply:
done